### DISCUSSION TITLE
How can you disable tqdm?

### DISCUSSION THREAD

**Discussion by fortyfourforty** [xhluca/bm25s#85] [Q&A] [answered]
Is there a way to disable tqdm in bm25s?
I need to extract tons of queries and the tqdm is annoying.

Tried the `os.environ["TQDM_DISABLE"] = "1"` but it does not work.

Thanks.

**Comment by xhluca** [comment, marked as answer]
if you uninstall tqdm it will not use it. you can also set `show_progress=False` in retrieve to disable it for a given retrieval: https://github.com/xhluca/bm25s/blob/f161fba91753fe25770614cc036d6502dce37c26/bm25s/__init__.py#L612-L614

Similarly for `index` you can use `show_progress=False`: https://github.com/xhluca/bm25s/blob/f161fba91753fe25770614cc036d6502dce37c26/bm25s/__init__.py#L412
